podcast/2.7.md

```diff
@@ -0,0 +1,717 @@
+
+**Beyang Liu:**
+Yeah, that's awesome. So walk us through the story from those beginnings to the beginnings of Unleash, what's the progression of your journey as programmers leading up to that point?
+
+**Egil Østhus:**
+Yeah. I think at some points, I figure out I was working at the company called Tandberg, acquired by Cisco, and I was fortunate to work with really skilled developers. I had this developer, actually his core duty was to take the C programming language after it was compiled to whatever assembly code or a machine code and optimize it even further. And at that point I just, yeah, understood that I'm never going to be at that level.
+
+**Beyang Liu:**
+So he would hand optimize the outputted assembly?
+
+**Egil Østhus:**
+Yeah.
+
+**Beyang Liu:**
+Wow.
+
+**Egil Østhus:**
+Yeah. And of course this was at the time where I assumed this wasn't wrong. This was the first attempt to really make video conferencing run on an internet processor. That was a really, really tough challenge back in the day. But my core focus was realizing I'm never going to be this good. I'm pretty good at all this stuff. I'm pretty good at making other people shine and get good at what they're good at, and I moved into more management stuff and luckily we have guys like Ivar still around that actually know how to do some proper coding.
+
+**Beyang Liu:**
+Cool. Ivar, what about you? What's been your journey?
+
+**Ivar Østhus:**
+Yeah, so I started out from the university. I started out as a consultant, as you often do for some reason. I don't know, that's at least very common in Norway. And I worked in the bank sector for a couple of years, banking finance. And I found it a bit depressing to be honest, because it was, maybe we were allowed to release every month. It was just terrible working like this. And I started looking for other opportunities and at some point I was lucky enough to start in this company called FINN. And we will talk a bit more about that later, I guess. And there I've been on a very interesting journey and people were much more lean and agile and wanted to really make impact on the code I wrote and making sure that I actually sold a user need and getting it out there as soon as possible to the users.
+
+**Ivar Østhus:**
+So this was really an opportunity for me to learn a lot, to be honest and in FINN, I have taken many, many different positions. I started out as a consultant actually, but I soon realized that I just wanted to work there full time. So I switched and started out as a developer, and I worked up the ranks. I started to become a tech lead. And then I started as an architect and at some point I also got the chief enterprise architect leading architect with FINN.
```

Comment:
"at some point I also got the chief enterprise architect leading architect with FINN." -> Should the second "architect" here be "architecture?" Basically, is there a readability issue here?